src/app.ts

```diff
@@ -1,12 +1,14 @@
-import express, {Application} from "express";
-import { createServer  } from "node:http";
-import { User } from "./models/user.model";
-import { apiRouter }  from "./routes/useRoutes";
-import cors from "cors";
-import { sequelize, testConnection } from "./db/sequelizeConfig";
-import { SocialNetwork } from "./models/socialnetwork.model";
-import { logger } from "./middlewares/logger.middleware";
+import express, { Application } from 'express';
```

Comment:
Pourquoi t'utilise createServer en plus d'Express ?

Reply:
C'est un reste des tests avec les WebSockets

Reply:
Ah c'est noté